dev-docs/bidders/yieldlove

```diff
@@ -0,0 +1,19 @@
+---
+layout: bidder
+title: Yieldlove
+description: Yieldlove adapter
+biddercode: yieldlove
+pbjs: true
+gvl_id: 251
+gdpr_supported: true
+sidebarType: 1
+---
+
```

Comment:
Usually bid adapters have a small notice on who to contact, e.g.

> The Adagio bidder adapter requires setup and approval from the Adagio team. Please reach out to contact@adagio.io for more information.

from https://docs.prebid.org/dev-docs/bidders/adagio.html